### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by anilpdl**

### CODE REVIEW

File: dist-diamonds.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+for(var i=0;i<diamonds.length;i++){
+	total=total+diamonds[i];		//Total array values
+}
+
+if(total%personNum!=0){
+	console.log("Cannot distribute equally to "+personNum+" people");
+	return;
+}
+
+diamonds.sort();			//Sorts array values low-high
```
Comment (poudelprakash) [inline dist-diamonds.js:23]: you can chain methods:
`diamonds.sort().reverse()`

File: dist-diamonds.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+diamonds.reverse();			// Reverse array values
+
+for(i=0;i<diamonds.length;i++){
+	diamondSum = diamondSum + diamonds[i];
```
Comment (poudelprakash) [inline dist-diamonds.js:27]: where is diamondSum defined?

Comment (anilpdl) [reply dist-diamonds.js:27]: line:10
along with total

File: dist-diamonds.js (hunk 1 above)
Comment (anilpdl) [reply dist-diamonds.js:23]: ok

File: dist-diamonds.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+/* Write a program which has an array of diamonds with different value e.g. [1,2,3,4,5]. 
+The program should distribute the diamonds equally to 3 people. For example: Person 1: 1,4 Person 2: 2,3 Person 3: 5 
+There may be any number of diamonds with any value. If diamonds cannot be equally distributed(e.g [1,2,3,4]), 
+the program should print message “Cannot distribute equally to 3 people”. */
+
+
+
+var diamonds = [1,1,1,1,1,1,1,2,2,1];
+var itemsToSplice = [];  //For storing values that are to be distributed
+var total=0, personNum=3,diamondSum=0,currentPerson=1;
```
Comment (poudelprakash) [inline dist-diamonds.js:10]: always have space around operators,
should be `total = 0` instead of `total=0`

File: dist-diamonds.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+The program should distribute the diamonds equally to 3 people. For example: Person 1: 1,4 Person 2: 2,3 Person 3: 5 
+There may be any number of diamonds with any value. If diamonds cannot be equally distributed(e.g [1,2,3,4]), 
+the program should print message “Cannot distribute equally to 3 people”. */
+
+
+
+var diamonds = [1,1,1,1,1,1,1,2,2,1];
+var itemsToSplice = [];  //For storing values that are to be distributed
+var total=0, personNum=3,diamondSum=0,currentPerson=1;
+var getDiamondNum = ":";	// For printing distributed value
```
Comment (poudelprakash) [inline dist-diamonds.js:11]: you don't need to store this in a variable

**Comment by poudelprakash** [comment]
@pdlanl please push latest update on this branch

**Pull request merged**